test/corpus/custom_types.txt

```diff
@@ -0,0 +1,443 @@
+================================================================================
+CREATE TYPE
+================================================================================
+
+ CREATE TYPE sports
+
+--------------------------------------------------------------------------------
+
+(program
+  (statement
+    (create_type
+      (keyword_create)
+      (keyword_type)
+      (identifier))))
+
+================================================================================
+CREATE TYPE ENUM
+================================================================================
+
+CREATE TYPE colors as ENUM (
+    'red',
+    'green',
+    'blue'
+);
+
+--------------------------------------------------------------------------------
+
+(program
+  (statement
+    (create_type
+      (keyword_create)
+      (keyword_type)
+      (identifier)
+      (keyword_as)
+      (keyword_enum)
+      (literal)
+      (literal)
+      (literal))))
```

Comment:
seeing this inline I wonder if it'd be useful to have a containing expression for the values -- is filtering for `literal`s specific enough?

Reply:
There are no field identifiers shown in the test. If you parse it with tree-sitter directly, this looks like this:

```
(program [0, 0] - [5, 0]
  (statement [0, 0] - [4, 1]
    (create_type [0, 0] - [4, 1]
      (keyword_create [0, 0] - [0, 6])
      (keyword_type [0, 7] - [0, 11])
      (identifier [0, 12] - [0, 18])
      (keyword_as [0, 19] - [0, 21])
      (keyword_enum [0, 22] - [0, 26])
      enum_element: (literal [1, 4] - [1, 9])
      enum_element: (literal [2, 4] - [2, 11])
      enum_element: (literal [3, 4] - [3, 10]))))
```

I will make it an extra node, so it will be clearer.